code/game/machinery/bots/ed209bot.dm

```diff
@@ -1017,6 +1017,8 @@ Auto Patrol: []"},
 				var/turf/T = get_turf(user)
 				to_chat(user, "<span class='notice'>You start to wire [src]...</span>")
 				sleep(40)
+				if(build_step != 6)
+					return
 				if(get_turf(user) == T)
 					coil.use(1)
 					build_step++
```

Comment:
Well partially because if this was done poorly you could modify the buildstep many times at once, setting the build set to a specific value is more sane in that sense